src/__tests__/Registration.test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducers from '../redux/reducers/index';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+// import  RegistrationPage  from '../components/pages/Registration/RegisterStep1';
```

Comment:
Hey guys, the loom video was pretty clean and explanatory, why is this line commented out though?